src/Parsers/HiliteComparator/CMakeLists.txt

```diff
@@ -5,3 +5,6 @@ add_headers_and_sources(hilite_comparator .)
 add_library(hilite_comparator ${hilite_comparator_sources})
 
 target_link_libraries(hilite_comparator PRIVATE dbms)
+
+clickhouse_add_executable(hilite_comparator_main hilite_comparator_main.cpp)
+target_link_libraries(hilite_comparator_main PRIVATE hilite_comparator)
```

Comment:
"main" is not a clear name. I suggest renaming it into just "hilite_comparator" and the library into "hilite_comparator_library".